Library/Formula/gptfdisk.rb

```diff
@@ -13,10 +12,15 @@ class Gptfdisk < Formula
     sha256 "95593d9ce977a9529b11c9de8ee1089e56c67d76a642e13bdac31097aa5c7f69" => :mountain_lion
   end
 
+  # Due to SIP
+  depends_on MaximumMacOSRequirement => :yosemite
+
   depends_on "popt"
   depends_on "icu4c"
 
   def install
+    inreplace "Makefile.mac", "/opt/local/lib/libncurses.a", "/usr/lib/libncurses.dylib"
```

Comment:
I doubt this is worth reporting upstream since we'll probably have to boneyard the whole thing soon.